omglol/paste_test.go

```diff
@@ -0,0 +1,90 @@
+package omglol
+
+import (
+	"testing"
+)
+
+func validatePaste(t *testing.T, p Paste) {
+	if len(p.Title) <= 0 {
+		t.Error("Paste Title is empty.")
+	}
+	if len(p.Content) <= 0 {
+		t.Error("Paste Content is empty.")
+	}
```

Comment:
An empty paste is actually valid. We can check to make sure that the timestamp is being set though. 
```suggestion
	if p.ModifiedOn <= 0 {
		t.Error("ModifiedOn time is unset.")
	}
```